BoxContentSDK/BoxContentSDK/Models/BOXUser.m

```diff
@@ -114,6 +114,15 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
             self.avatarURL = [NSURL URLWithString:avatarURLString];
         }
         
+        self.hasCustomAvatar = BOXAPIBooleanUnknown;
+        NSNumber *hasCustomAvatarNumber = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyHasCustomAvatar
```

Comment:
is this API automatically returned by the API ? or do we have to specify all fields to get it ?